application/actions/requests.go

```diff
@@ -106,25 +111,15 @@ func convertRequest(ctx context.Context, request models.Request) (api.Request, e
 	tx := models.Tx(ctx)
 	user := models.CurrentUser(ctx)
 
-	createdBy, err := loadRequestCreatedBy(ctx, request)
+	createdBy, err := convertUser(ctx, request.CreatedBy)
 	if err != nil {
 		return api.Request{}, err
 	}
 	output.CreatedBy = createdBy
 
-	origin, err := loadRequestOrigin(ctx, request)
-	if err != nil {
-		return api.Request{}, err
-	}
-	output.Origin = origin
+	output.Origin = convertRequestOrigin(request)
```

Comment:
The `convertToOtherType` on the parent object handles the destination. Unfortunately, it's the only child object that is pain-free like that. Others have an ID that needs to be moved from `uuid` to `id` and other transformations. So, do you think I should make the conversion explicit just to avoid this kind of ambiguity?

Reply:
If destination is the only object handled by convertToOtherType, I feel like an explicit conversion would be better for clarity and consistency, even if it is somewhat redundant.